app/AppDelegate.m

```diff
@@ -65,21 +65,35 @@ - (int)startThings {
     exit_hook = ios_handle_exit;
     
     // configure dns
-    struct __res_state res;
-    err = res_ninit(&res);
-    NSMutableString *resolvConf = [NSMutableString new];
-    for (int i = 0; res.dnsrch[i] != NULL; i++) {
-        [resolvConf appendFormat:@"search %s\n", res.dnsrch[i]];
+    res_state _state;
+    _state = malloc(sizeof(struct __res_state));
```

Comment:
I don't think malloc/free is really necessary here, the res_state can still be stack allocated like it was before.

Reply:
Good point. Fixed